### DISCUSSION TITLE
Conflict AutoDrive 1.1.1.4 with mod Tardis Teleportation V 0.9.1.3

### DISCUSSION THREAD

**Discussion by AlexBig101** [Stephan-S/FS19_AutoDrive#2022] [General]
The new FS19_AutoDrive 1.1.1.4 mod has problems with the teleport mod. When I teleport in a vehicle, it blocks her camera. If you just transfer to another vehicle, then everything is fine there, but as soon as she teleports, her camera will also be blocked.
This is the first time I use the AutoDrive mod, I immediately downloaded the latest version 1.1.1.4. As with previous versions, I do not know.
Tried on New save on standard map with only two mods Tardis Teleportation V 0.9.1.3 and FS19_AutoDrive 1.1.1.4, and the same problem.
Can you help ???

**Comment by Dzi4d3k** [comment]
That been fixed in:
https://github.com/Stephan-S/FS19_AutoDrive/issues/1992
you should go and grab newer version of Tardis Teleport mod::
https://github.com/sperrgebiet/FS19_Tardis

**Reply by AlexBig101** [reply]
Thank you so much. Now FS19_Tardis 0.9.2.1 and FS19_AutoDrive 1.1.1.4 does not have the problem I wrote about.
